packages/core/src/base-axis-chart.ts

```diff
@@ -177,6 +181,22 @@ export class BaseAxisChart extends BaseChart {
 		};
 	}
 
+	/**
+	 * Adds a resize observer to the x and y grid and resizes the backdrop on change
+	 */
+	gridResizeObserver() {
+		const gridWidth = this.holder.querySelector(".y.grid");
+		const gridHeight = this.holder.querySelector(".x.grid");
+
+		const resizeObserver = new ResizeObserver((entries) => {
+			this.drawBackdrop();
+		});
+
+		// watch resizing of both grids
+		resizeObserver.observe(gridWidth);
+		resizeObserver.observe(gridHeight);
+	}
```

Comment:
we already catch resize events for the entire chart. Seems like in `resizeChart()` we can just call `this.drawBackdrop();`?

Reply:
to my understanding (correct me if im wrong), the reason backdrop isnt resizing fluidly using drawBackdrop() within the resizeChart() is because theres is a timeout in updateXandYGrid(). this is since there is some delay with the axis repositioning (of ticks and stuff). having the resize observer on the actual grid will force the backdrop to resize if the grid is to ever resize, rather than if the entire chart receives a resize event (and is not even done resizing). 

the main issue is that the current resizeObserver in base-chart calls resizeChart and all the calculations are done so quick that the x and y grid are not yet in their final locations causing backdrop to render improperly. if the backdrop is tied to the actual x and y grid on resize, then there is a valid position for the backdrop to when it is called after grid has been updated rather than in preparation of an update.

Reply:
is there any way to do this without adding a second resizeObserver?